src/test/suite/fixtures/recorded/actions/cloneVest2.yml

```diff
@@ -0,0 +1,31 @@
+languageId: plaintext
```

Comment:
Not sure how this one differs from the other "clone vest"

Reply:
See above